static/js/pages/independent-expenditures.js

```diff
@@ -46,8 +46,9 @@ var columns = [
       }
     }
   },
-  tables.dateColumn({data: 'expenditure_date', className: 'min-tablet hide-panel-tablet'}),
-  tables.urlColumn('pdf_url', {data: 'expenditure_description', className: 'all hide-panel', orderable: false}),
+  _.extend(columns.supportOpposeColumn, {className: 'min-tablet'}),
```

Comment:
This line mutates `columns.supportOpposeColumn`--if any code uses that column later on, it'll still have the added class name. That's probably fine in this case, but it's generally good practice to use something like `_.extend({}, columns.supportOpposeColumn, {className: 'min-tablet'})` to avoid mutating the original object.